autonormalize/autonormalize.py

```diff
@@ -92,15 +92,22 @@ def make_entityset(df, dependencies, name=None, time_index=None):
 
     while stack != []:
         current = stack.pop()
+        if (current.df.ww.schema is None):
+            current.df.ww.init(index=current.index[0], name=current.index[0])
+
+        current_df_name = current.df.ww.name
         if time_index in current.df.columns:
-            entities[current.index[0]] = (current.df, current.index[0], time_index)
+            entities[current_df_name] = (current.df, current.index[0], time_index)
```

Comment:
As we have updated other libraries, we have moved away from using the entity/entities terminology in favor of dataframe/dataframes. If there are just a handful of places to update, maybe we should do that now, but if there are a lot that need to be changed, we can create a new issue and work in it separately.

Reply:
Done! It wasn't too bad